src/main/java/mate/jdbc/dao/CarDao.java

```diff
@@ -0,0 +1,18 @@
+package mate.jdbc.dao;
+
+import java.util.List;
+import mate.jdbc.model.Car;
+
+public interface CarDao {
+    Car create(Car car);
+
+    Car get(Long id);
```

Comment:
It's better to return optional here, because Your differenet services may process the case when obejct is absent differently. That's the default behaviour, that is implemented the same way in Spring Data. 
Besides returning null is a bad practice